src/Symfony/Component/Notifier/Bridge/GatewayAPI/GatewayAPITransport.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Bridge\GatewayAPI;
+
+use Symfony\Component\Notifier\Exception\LogicException;
+use Symfony\Component\Notifier\Exception\TransportException;
+use Symfony\Component\Notifier\Message\MessageInterface;
+use Symfony\Component\Notifier\Message\SentMessage;
+use Symfony\Component\Notifier\Message\SmsMessage;
+use Symfony\Component\Notifier\Transport\AbstractTransport;
+use Symfony\Contracts\EventDispatcher\EventDispatcherInterface;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+
+/**
+ * @author Piergiuseppe Longo <piergiuseppe.longo@gmail.com>
+ * @experimental in 5.2
+ */
+final class GatewayAPITransport extends AbstractTransport
+{
+    protected const HOST = 'gatewayapi.com';
+
+    private $authToken;
```

Comment:
I am not sure, do other notifier use property types?

Reply:
You are talking about hosts? If yes, it's declared in other transports like FreeMobileTransport, SmsapiTransport, NexmoTransport, etc.

Reply:
No I mean if they use `private string $authToken` or not

Reply:
Some Transport use it, for instance, `SmsapiTransport`.

Reply:
Symfony 5.x supports PHP 7.2+ so we cannot use typed properties.